src/components/index-page/IndexLaunched.tsx

```diff
@@ -4,78 +4,76 @@ import React from "react";
 import arrowForward from "../../assets/images/arrow_forward.png";
 import "./css/index-launched.css";
 
-export const IndexLaunched = () => {
-  return (
-    <div className="index-launched">
-      <div className="index-launched__contents">
-        <div className="index-launched__heading-container">
-          <h2 className="index-launched__heading">Go from stuck and stalled out to LAUNCHED</h2>
-        </div>
-        <div className="index-launched__columns-container">
-          <div className="index-launched__column">
-            <div className="index-launched__image-container">
-              <img className="index-launched__image" src="https://placehold.co/378x330" />
-            </div>
-            <h2 className="index-launched__column-heading">
-              Get a proven and transparent plan to take you where you want to go
-            </h2>
-            <p className="index-launched__text">
-              You have a vision, and you need a partnership that can bring a clear path to get your
-              software launched.
-            </p>
-            <p className="index-launched__text">
-              Get the clarity you need, every step of the way, with a partner that prioritizes
-              communicating.
-            </p>
-            <p className="index-launched__text">
-              <Link className="index-launched__learn-more" to="#">
-                <span>Learn More</span>
-                <img className="index-launched__arrow-forward" src={arrowForward} />
-              </Link>
-            </p>
+export const IndexLaunched = () => (
+  <section className="index-launched">
+    <div className="index-launched__contents">
+      <div className="index-launched__heading-container">
+        <h2 className="index-launched__heading">Go from stuck and stalled out to LAUNCHED</h2>
```

Comment:
you can use a `<span>` with `text-transform: uppercase` here so that the markup reads `launched`